Prac07/GuitaerClass.py

```diff
@@ -0,0 +1,18 @@
+class GuitarDetails:
+    def __init__(self, guitar_name=" ", year_made=0, price=0):
+        self.guitar_name = guitar_name
+        self.year_made = year_made
+        self.price = price
+
+    def __str__(self):
+        return print("Guitar {}: {} {}".format(self.guitar_name, self.year_made, self.price))
```

Comment:
To get the guitars to print correctly, you need to remove the `print` from the `__str__`. Replace the line with:
`return "Guitar {}: {} {}".format(self.guitar_name, self.year_made, self.price)`
Everything else seems to look good, however I'd watch your spelling with the word "guitar" just for consistency's sake. Easily remedied using the Refactor method in PyCharm.